include/irr/asset/IAssetWriter.h

```diff
@@ -24,15 +24,30 @@ enum E_WRITER_FLAGS : uint32_t
     EWF_BINARY = 1u<<2u
 };
 
+//! Parameter flags for a loader
+/**
+	These are extra flags that have an impact on extraordinary tasks while writing to file.
+	E_LOADER_PARAMETER_FLAGS::ELPF_NONE is default and means that there is nothing to perform.
+	E_LOADER_PARAMETER_FLAGS::ELPF_RIGHT_HANDED_MESHES specifies that a mesh will be flipped, 
+	if current flag responsible for orientation is signed as left handed. If it isn't,
+	no flipping will be performed, because there will be no need for that.
+*/
+
+enum E_WRITER_PARAMETER_FLAGS : uint64_t
+{
+	EWPF_NONE = 0,											//!< default value, it doesn't do anything
+	EWPF_RIGHT_HANDED_MESHES = 0x1,							//!< specifies that a mesh will be flipped in such a way that it'll look correctly in right-handed camera system						
```

Comment:
call it `EWPF_MESH_IS_RIGHT_HANDED`

It does not mean it will be flipped.

Explanation:

If a mesh format is right-handed then no flipping for right handed input mesh, but flipping for left handed input mesh (the default).
Example: Possibly OBJ (I think we flip when we load by default), definitely `.gtlf`

If a mesh format is left-handed then flip for right-handed input but no flipping for left-handed input.
Example: DirectX .X

Reply:
btw move the enum inside `IAssetWriter`